lib.php

```diff
@@ -124,20 +124,14 @@ function local_augmented_teacher_send_reminder_message() {
 
     cron_setup_user(null, null, true);
 
-    $sql = "SELECT rem.*, cm.completionexpected,
-                   case rem.type
-                       when ".REMINDER_BEFORE_DUE." then cm.completionexpected - rem.timeinterval
-                       when ".REMINDER_AFTER_DUE." then cm.completionexpected + rem.timeinterval
-                   end as scheduled
-              FROM {local_augmented_teacher_rem} rem
-              JOIN {course_modules} cm
-                ON rem.cmid = cm.id
-             WHERE rem.deleted = ?
-               AND rem.enabled = ?
-               AND rem.messagetype = ?";
-
-    $rs = $DB->get_recordset_sql($sql, array(0, 1, MESAGE_TYPE_REMINDER));
-    foreach ($rs as $reminder) {
+    $reminders_repo = \local_augmented_teacher\reminders_factory::get_repository($DB);
+    $reminders = $reminders_repo->get_activity_reminders();
+
+    $user_enrolments_repo = \local_augmented_teacher\user_enrolments_factory::get_repository($DB);
```

Comment:
Same as above. Same applies to all other places where $DB is passed as an argument.